src/services/public/images-service.js

```diff
@@ -30,26 +30,69 @@ module.exports = {
     });
   },
 
+  /**
+   * Gets all Image object records in the Images table.
+   * @param {Request} req Request entity for getting data from it.
+   * @returns {Array|Object}} Returns the responce with all Image objects from the Images table and number of records.
+   **/
+  async getPublicPagination(req) {
+    const { page, limit } = req.query;
+
+    return imagesModel.findAndCountAll({
+      where: {
+        isPrivate: false,
+      },
+      offset: (page - 1) * limit,
+      limit: limit,
+      subQuery: false,
+    });
+  },
+
   /**
    * Gets all Image object records in the Images table with one owner.
-   * @param {Integer} userId User id for search.
+   * @param {Request} req Request entity for getting data from it.
    * @returns {Array|Object} Returns the responce with all Image objects from the Images table.
    **/
-  async getAllByUserId(userId) {
+  async getAllByUserId(req) {
+    const userId = req.body.userId;
+
     return imagesModel.findAll({
       where: {
         userId,
       },
     });
   },
 
+  /**
+   * Gets all Image object records in the Images table with one owner.
+   * @param {Request} req Request entity for getting data from it.
+   * @returns {Array|Object} Returns the responce with all Image objects from the Images table.
+   **/
+  async getAllByUserIdPagination(req) {
+    //const userId = req.body.userId;
+    let userId;
+    req.body.userId ? (userId = req.body.userId) : (userId = 1);
```

Comment:
Do not use such logic, it's strange.
Replace that by smth like:
const userId = req.body.userId ? req.body.userId : 1

Reply:
But what that func do at all ?